templates/index.html

```diff
@@ -7,30 +7,30 @@
 
 {% block content %}
   <section class="hero">
-    {% set boskonf_live = config.extra.boskonf_live %}
-
-    {% if boskonf_live %}
-      <style>
-        [data-theme="dark"] #banner {
-          background: url({{get_url(path="/img/boskonf_banner_dark.svg") | safe }}) center / cover;
-        }
-        #banner {
-          width: 100%;
-          height: 350px;
-          display: block;
-          position: relative;
-          background: url({{get_url(path="/img/boskonf_banner.svg") | safe }}) center / cover no-repeat;
-          background-size: contain;
-        }
-      </style>
-      <div id="banner"></div>
+    {% set custom_hero = config.extra.custom_hero %}
 
+    {% if custom_hero %}
+      <section class="pyro-container">
+        <div class="pyro">
+          <div class="before"></div>
+          <div class="after"></div>
+        </div>
+      </section>
```

Comment:
En `section` må ha en heading inn i seg for å være "gyldig" semtantisk HTML:
https://www.w3.org/WAI/WCAG21/Understanding/section-headings.html

Siden det kun er dekorativt så fungerer `div` helt fint her 😊 
```suggestion
      <div class="pyro-container">
        <div class="pyro">
          <div class="before"></div>
          <div class="after"></div>
        </div>
      </div>
```